crates/biome_service/tests/invalid/hooks_incorrect_options.json.snap.new

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I don't think this file should be here

Reply:
@ematipico  oh really? Seemed right because it is a new option added to the nursery which maybe the tests needs to be aware of?

Reply:
Probably a mistake when you resolved the conflicts. Those are old files that aren't preset anymore